horovod/common/operations.cc

```diff
@@ -1109,6 +1158,37 @@ void BackgroundThreadLoop(HorovodGlobalState& state, MPIContext& ctx) {
     state.message_table = std::unique_ptr<MessageTable>(new MessageTable());
   }
 
+  state.cpu_operation = HOROVOD_MPI;
+
+#if HAVE_GLOO
+  state.cpu_operation = HOROVOD_GLOO;
```

Comment:
We should remove this - Gloo should never be the default at this point.  It would only be default if opted in at compilation below.

Reply:
Yes.